src/en/index.html

```diff
@@ -36,7 +36,7 @@ <h2 class="h2">Reliable and scalable storage designed for any organization</h2>
       </div>
       <div>
         <h2 class="h2">Ceph for developers</h2>
-        <p class="p mb-8">Setup, deploy, manage, code and contribute to Ceph.</p>
+        <p class="p mb-8">Deploy Ceph now. Use the links below to acquire Ceph and deploy a Ceph cluster.</p>
```

Comment:
For this, I think we should

1. add a separate block above that says "Deploy or manage a Ceph cluster" and has the 'getting started' and 'documentation' links
2. remove the getting started link from this block, and change text to "Study, modify, and contribute to Ceph" or similar and keeps the code and contribute links

Reply:
I've made these alterations.

However, as foretold by (my own) prophecy, I messed up the formatting of the buttons. Here's where Pete comes to the rescue.

Pete, can you make sense of the nonsense of buttons I have foisted upon you?

Reply:
@Pete-Robelou see supra

Reply:
Yeah, all good. I'll sort it.

Reply:
@zdover23 @liewegas I've updated the layout with a new section, capitalised the text and amended a few of the links.

**Deploy or manage a Ceph cluster**
- Deploy or manage a Ceph cluster > `/users/`
- Getting started > `/users/getting-started/`
- Documentation > `/users/documentation/`

**Ceph for developers**
- Study, modify and contribute > `/users/getting-started/` Should this link to `/developers/` ?
- Code > `/developers/code/`
- Contribute > `/developers/contribute/`

Reply:
yeah, ``/developers`` I think

Reply:
I've updated Study, modify and contribute to `/developers/`